wolfcrypt/src/asn.c

```diff
@@ -21875,6 +21875,19 @@ static int DecodeCertInternal(DecodedCert* cert, int verify, int* criticalExt,
         /* Set fields extracted from data. */
         cert->version = version;
         cert->serialSz = (int)serialSz;
+
+    #if !defined(WOLFSSL_NO_ASN_STRICT) && !defined(WOLFSSL_PYTHON)
```

Comment:
I understand the no asn struct, but why the `WOLFSSL_PYTHON`? Does it specifically use a serial no 0 for testing?

Reply:
Exactly, it appears that one of the test certificates has a serial number of 0.

```
Python-3.8.14/Lib/test$ wolfssl x509 -serial -noout -in nullbytecert.pem
serial=00
```

Removing parsing of serial number 0 results in the equivalent "make check" failing with Python.